### PR TITLE
Add Drivers License Input component.

### PR DESCRIPTION
**Pull request by karthik-sista**
[CL-125](https://osi-cwds.atlassian.net/browse/CL-125)

### CODE REVIEW

File: apps/www/src/modules/components/articles/Forms/examples/DriversLicenseExample.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { DriversLicenseInput } from '@cwds/forms'
+
+const DriversLicenseExample = () => {
+  return (
+    <div>
```
Comment (mlipardo) [inline apps/www/src/modules/components/articles/Forms/examples/DriversLicenseExample.jsx]:
```suggestion
    remove <div>
```

Comment (mlipardo) [inline apps/www/src/modules/components/articles/Forms/examples/DriversLicenseExample.jsx]:
```suggestion
    <div>
```

Comment (mlipardo) [inline apps/www/src/modules/components/articles/Forms/examples/DriversLicenseExample.jsx]:
```suggestion

```

Comment (mlipardo) [reply apps/www/src/modules/components/articles/Forms/examples/DriversLicenseExample.jsx]: remove the `div`

File: libs/forms/src/DriversLicenseInput.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
```
Comment (eckdanny-osi) [inline libs/forms/src/DriversLicenseInput.tsx]: we don't want to ship a "Driver's License Input" component (not now anyway) unless we can stand behind the formatting rules and such for _all_ possible values. 

For this PR, plz just demonstrate "How one would compose primitive widget types to create a compound field"

(There should not be a new component in `@cwds/forms` as a result)

File: libs/forms/src/StatesData.ts [hunk 3]
```diff
@@ -0,0 +1,238 @@
+export const StatesData = [
```
Comment (eckdanny-osi) [inline libs/forms/src/StatesData.ts]: ditto ^

File: apps/www/src/App.js [hunk 4]
```diff
@@ -18,6 +18,7 @@ import MessageArea from './demos/MesssageArea'
 import PageTemplates from './examples/PageTemplates'
 import CardHeaderDemo from './demos/CardHeaders'
 import ToggleDemos from './demos/ToggleDemos'
+import DriversLicenseInput from './demos/DriversLicense'
```
Comment (mlipardo) [inline apps/www/src/App.js]: Can we be consistent with naming since we started with Demos.

Comment (mlipardo) [inline apps/www/src/App.js]:
```suggestion
import DriversLicenseDemo from './demos/DriversLicense'
```

**Comment by mlipardo** [comment]
Change the title also to **Add Drivers License Input Demo**.

**Pull request merged**